app/services/service_pattern.rb

```diff
@@ -14,7 +14,7 @@ module ClassMethods
     def call(**args)
       return new.call if args.empty?
```

Comment:
Same here re: forwarding, then we can bin that first check all together

Reply:
Done